wiki/Tournaments/SOFT/6/en.md

```diff
@@ -0,0 +1,189 @@
+- Players who are in the losers bracket at the start of the Quarterfinals or further are required to play in two matches in a single stage. However, players who lose their first match in the losers bracket in those stages will no longer need to play in any further matches, for they are eliminated from the tournament.
+
+### Tournament play
+
+#### Protocol
+
+- Overlaps between matches may happen in certain stages. As a result, streaming every match is not viable. Tournament staff will do their best to give every player in the tourney some stream time.
+- Your matches will be listed in schedules. **It is your duty as a player to properly identify your timeslot and reschedule your match with your opponent as soon as possible if a reschedule is necessary.**
+- Reschedules will only be considered if both players agree to a time. This needs to be done and notified to the tournament staff before **Wednesday at 23:59 UTC+0** during the particular week when your match takes place.
+- ***It is your right as a player and tournament entrant to expect that staff will perform to the best of their abilities in moderating matches in such a way that they will conclude in a timely manner.***
+  - Rooms with matches will be created before the match start time, and you will be invited to the corresponding tournament room before the match start time.
+  - For maximal accountability on the SOFT6 staff, each match will have a primary referee and a backup referee. That way, you will be assured that someone will be present to handle your tournament match.
+  - If both players arrive on time, tournament proceedings will begin promptly at the start time of the match, **no earlier and no later.**
+  - If your match starts in less than five (5) minutes and you have not been invited to your corresponding multiplayer room, notify any Tournament Staff member who is online on Discord or in-game that you are a player in SOFT6 and you currently do not have an invitation to your match. They will alleviate the situation by notifying the designated referees for your match to give you an invitation to your corresponding multiplayer room.
+- ***It is our right as tournament staff to expect that you will arrive for your matches before your expected time.***
+  - Players will be given a grace period of ten (10) minutes after the match start time to enter their room. If they have not entered by this point, the match will be forfeited and the player that has arrived will win by default. However, exceptions will be given to players who were late due to technical issues/staff impropriety.
+    - **Players who enter the room during the grace period cannot select a warm-up for the match.**
+  - In the extreme circumstance where both players do not show, **the player with the higher group stage seed in the match-up will advance automatically.** During Playoff Stage, **the player with the higher qualifier seed in the match-up will advance automatically.**
```

Comment:
to clarify, the period is included if the whole sentence is bolded/italicised, but not if it's only part of a sentence. it's a style thing and not very noticeable in this case but good to keep in mind. (note that the same thing applies for parentheses and links for example.)

```suggestion
  - In the extreme circumstance where both players do not show, **the player with the higher group stage seed in the match-up will advance automatically**. During Playoff Stage, **the player with the higher qualifier seed in the match-up will advance automatically**.
```